src/type-factory.ts

```diff
@@ -118,4 +122,52 @@ export class TypeFactory<T> {
     static sample<P>(iterable: Iterable<P>): Generator<P, P, P> {
         return sample<P>(iterable);
     }
+
+    async write(path: string, options?: FactoryBuildOptions<T>): Promise<T> {
+        const filename = fileName(path);
+        const file = fileExists<T>(filename);
+        if (file) return file;
+
+        const result = await this.build(options);
+        fs.writeFile(path, JSON.stringify(result), fileError);
```

Comment:
we should note in the docs that the various fixture based methods do not support generating objects with function properties - also some other stuff. 

JSON.stringify/parse is not able to stringify and parse all types of data, e.g. it cant handle blobs, functions and a few other types. But i think this is fine otherwise.